umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/grid-layout/README.md

```diff
@@ -1,4 +1,8 @@
-# Grid Layout
+# Grid Layout (Legacy)
+
+{% hint style="info" %} We highly recommend that you use the [Block Grid](../../block-editor/block-grid-editor.md) instead.
+
+Grid Layout has been marked as obsolete and will be removed from future versions of Umbraco. {% endhint %}
```

Comment:
As we can't give a specific version number as to when we will remove the property, how about we write something like this instead:

```suggestion
Grid Layout has been marked as obsolete and development on the property editor has been discontinued. {% endhint %}
```